### PR TITLE
Added steps to change appliance TLS certs after deployment

### LINKED ISSUE #2424: Document how to replace appliance certificates after deployment
Per https://github.com/vmware/vic-product/issues/1810.

### PR DESCRIPTION
**Pull request by vidya-v**
Hi @stuclem ,

Request your review. 
Fixes #2424 

Thanks,
Vidya

### CODE REVIEW

File: docs/user_doc/vic_vsphere_admin/change_appliance_TLS_certs.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Change vSphere Integrated Containers Appliance TLS Certificates
+
+If you do not provide TLS certificates during vSphere Integrated Containers appliance deployment, the OVA installer uses auto-generated certificates. You can reconfigure the appliance settings to change the TLS certificates after you have deployed it.
+
+## Prerequisites
+
+Log in to the vSphere Client:
+
+- If you use vCenter Server 6.7 update 1 or later, you can use the HTML5 vSphere Client to reconfigure the appliance.
+- If you use a version of vCenter Server that pre-dates 6.7 update 1, you must use the Flex-based vSphere Web Client. You cannot reconfigure the appliance in the HTML5 vSphere Client .
```
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/change_appliance_TLS_certs.md]: "the HTML5 vSphere Client ." Stray space

File: docs/user_doc/vic_vsphere_admin/change_appliance_TLS_certs.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Procedure
+
+1. Shut down the vSphere Integrated Containers appliance by selecting **Shut Down Guest OS**.
+
+	**IMPORTANT**: Do not select **Power Off**.
+
+4. Right-click the vSphere Integrated Containers appliance VM, and select **Edit Settings**.
+5. In the **vApp Options** tab, reconfigure the appliance VM as necessary.
```
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/change_appliance_TLS_certs.md]: The way you get to the vApp options is different in Flex and H5. There is a description of both methods in https://vmware.github.io/vic-product/assets/files/html/latest/vic_vsphere_admin/reconfigure_appliance.html. This topic also mentions in passing that you can update the certificates, but does not explain exactly how, so maybe it's worth linking to this new topic from `reconfigure_appliance.md` too.

Comment (vidya-v) [reply docs/user_doc/vic_vsphere_admin/change_appliance_TLS_certs.md]: Thanks, @stuclem . I have made the changes.

**Comment by vidya-v** [comment]
Hi @luwang-vmware , can you please review these updates?
Thanks,
Vidya

**Comment by vidya-v** [comment]
Thank you, @luwang-vmware !

**Pull request merged**